src/ui/overlay/SearchOverlay.qml

```diff
@@ -25,19 +25,29 @@ import QtQuick 2.7
 import QtQuick.Layouts 1.1
 import QtQuick.Controls 2.0
 import Fluid.Controls 1.0
-import "../"
 
-ExpansionBarItem {
-    id: searchBarItem
+BaseOverlay {
+    id: searchOverlay
 
     property bool searchEnabled: true
 
     signal searchRequested(string text, bool backwards)
 
+    onOpened: {
+        searchField.forceActiveFocus();
+    }
+
+    implicitWidth: childrenRect.width
+
     RowLayout {
-        anchors.fill: parent
+        id: rowLayout
 
-        spacing: 8
+        anchors {
+            top: parent.top
+            bottom: parent.bottom
+        }
+
+        Item { implicitWidth: 16 }  // Spacer
 
         TextField {
             id: searchField
```

Comment:
clicking on the textfield doesn't give it focus, not sure what's going on maybe an issue with the overlay?